src/plugins/interactive-media/interactive-media-editor.js

```diff
@@ -19,6 +19,7 @@ import { sectionEditorProps } from '../../ui/default-prop-types.js';
 import StorageFilePicker from '../../components/storage-file-picker.js';
 import { Button, Form, Input, Radio, Spin, Switch, Tooltip } from 'antd';
 import ObjectMaxWidthSlider from '../../components/object-max-width-slider.js';
+import { ASPECT_RATIO } from '../../components/media-player-constants.js';
```

Comment:
We could move this `ASPECT_RATIO` to `./domain/constants.js` now.

Reply:
Yes, done (and also the other enum), I think they're both 'general' enough to live with the grown up constants.